style.css

```diff
@@ -440,3 +646,43 @@ a {
   color: #fff;
   border: 1px solid #fff;
 }
+
+@media screen and (min-width: 768px) {
+  /*  ||CONTACT SECTION|| */
+  .contact-section {
+    background-position: 25px 13px;
+  }
+
+  .div-contact {
+    width: 90%;
+    background-color: #6070ff;
+    justify-content: center;
+    padding: 2rem;
+    margin: 0 auto;
+  }
+
+  .contact-descr {
+    width: 100%;
+  }
+
+  .contact-form {
+    width: 90%;
+    justify-content: center;
+    padding: 2rem;
+    margin: 0 auto;
+    margin-top: -70px;
```

Comment:
- [ ] [**OPTIONAL**] This contact form should be center so it may stick to the Figma design :+1: 

![Screenshot from 2023-01-19 18-37-49](https://user-images.githubusercontent.com/80458199/213501230-dd2750e6-d97e-404c-802b-2e896f5bc591.png)

Reply:
I used margin:0 auto; to center it to the figma designs